### PR TITLE
EA-2631: Add standalone tests for response headers

### PR DESCRIPTION
**Pull request by ikattey**

### CODE REVIEW

File: entity-management-web/src/integration-test/java/eu/europeana/entitymanagement/web/BaseWebControllerTest.java [hunk 1]
```diff
@@ -39,6 +46,16 @@
     @Autowired
     private ObjectMapper objectMapper;
 
+    @Autowired
+    private WebApplicationContext webApplicationContext;
+
+    @BeforeEach
+    protected void setup() throws Exception {
+        this.mockMvc = MockMvcBuilders.webAppContextSetup(this.webApplicationContext).build();
```
Comment (gsergiu) [inline entity-management-web/src/integration-test/java/eu/europeana/entitymanagement/web/BaseWebControllerTest.java:54]: do we need the server initialization before each test run? This is quite an expensive operation...

File: entity-management-web/src/integration-test/java/eu/europeana/entitymanagement/web/EntityRegistrationIT.java [hunk 2]
```diff
@@ -82,78 +68,62 @@ void registerAgentStalinShouldBeSuccessful() throws Exception {
...
     }
 
-//    @Test
-    /* Doesn't work without correct zoho import configuration
-     * TODO: mock zoho response */
+    /**
+     * TODO: mock zoho response
+     */
+    @Test
+    @Disabled("Doesn't work without correct zoho import configuration")
```
Comment (gsergiu) [inline entity-management-web/src/integration-test/java/eu/europeana/entitymanagement/web/EntityRegistrationIT.java:87]: could we mock zoho responses as well?

**Review by gsergiu — COMMENTED** [review]
probably the mock for Zoho should be addressed in another ticket. Could we use @beforeAll for initialization of mockMvc?

File: entity-management-web/src/integration-test/java/eu/europeana/entitymanagement/web/EntityRegistrationIT.java (hunk 2 above)
Comment (ikattey) [reply entity-management-web/src/integration-test/java/eu/europeana/entitymanagement/web/EntityRegistrationIT.java:87]: Not in scope for this ticket.

File: entity-management-web/src/integration-test/java/eu/europeana/entitymanagement/web/BaseWebControllerTest.java (hunk 1 above)
Comment (ikattey) [reply entity-management-web/src/integration-test/java/eu/europeana/entitymanagement/web/BaseWebControllerTest.java:54]: `MockMvcBuilders.webAppContextSetup()` just creates a new MockMvc instance for each test, which is precisely what we want. It doesn't reload the web application context.

**Pull request merged**